helm/chart-operator-chart/templates/deployment.yaml

```diff
@@ -27,6 +27,7 @@ spec:
           - key: config.yaml
             path: config.yaml
       serviceAccountName: {{ .Values.name }}
+      priorityClassName: system-cluster-critical
```

Comment:
I used the same priority as Ingress Controller. As we want the operator running so it can apply changes.